test/integration/about_test.rb

```diff
@@ -14,31 +14,17 @@ class AboutIntegrationTest < IntegrationTestWithJavascript
 
   test "about page" do
     assert_index_page(about_index_path,"About", nil, false, false)
-    wait_for_ajax
```

Comment:
`axios` breaks `wait_for_ajax` (React components  use axios instead of Jquery)
it's redundant here.